frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -26,3 +28,10 @@ export const createLookup = <A extends K8sResourceKind>(
   }
   return {};
 };
+
+// To check if the provisioner is OCS based
+export const isCephProvisioner = (scProvisioner: string) => {
```

Comment:
This should go in a file specific to storage

Reply:
ack